app/models/deploy.rb

```diff
@@ -90,6 +92,24 @@ def variables
     stack.deploy_variables
   end
 
+  def refute!
```

Comment:
`reject` might be a better name for this. `refute` implies that there is some statement that you're saying is untrue (like `a == b` in the testing case).